components/backdrop/backdrop.js

```diff
@@ -209,12 +209,14 @@ function showAccessible(elems) {
 		} else {
 			elem.removeAttribute('role');
 		}
-		const ariaHidden = elem.getAttribute('d2l-backdrop-aria-hidden');
-		if (ariaHidden) {
-			elem.setAttribute('aria-hidden', ariaHidden);
-			elem.removeAttribute('d2l-backdrop-aria-hidden');
-		} else {
-			elem.removeAttribute('aria-hidden');
+		if (elem.nodeName === 'FORM' || elem.nodeName === 'A') {
+			const ariaHidden = elem.getAttribute('d2l-backdrop-aria-hidden');
```

Comment:
Should we use a const like `BACKDROP_ARIA_HIDDEN` to be consistent with `BACKDROP_ROLE`?

Reply:
Sure, yep, that's worthwhile.

Reply:
If we're making changes here, it'd probably be better to use `data-backdrop-aria-hidden`. Sprouting arbitrary invalid attribute names on native elements probably isn't great for HTML validation. (I'm going through a whole accessibility audit of our invalid HTML, so it's top of mind!)

Reply:
Cool, yeah, I was just about to switch that, as well as the one that's exported.